twill-yarn/src/main/java/org/apache/twill/internal/appmaster/RunnableProcessLauncher.java

```diff
@@ -76,6 +76,11 @@ public String toString() {
 
     return new ProcessController<R>() {
       @Override
+      public void close() throws Exception {
+        // no-op
```

Comment:
Controller ties to the lifecycle of the stuff that it is controlling. What do mean by leave the runnable process running when closing the controller? Is it related to #4 ?